husky_extensions/itembox.js

```diff
@@ -32,6 +32,8 @@ define(function() {
             hideSelectedCounter: false,
             defaultDisplayOption: 'top',
             actionIcon: 'fa-plus-circle',
+            publishedStateName: 'publishedState',
+            publishedName: 'published',
```

Comment:
I could use generic icon names here and override them in the internal link component.